src/NearOrg/CookiePolicy.jsx

```diff
@@ -117,6 +117,17 @@ return (
             </>
           ),
         },
+        {
+          value: "6",
+          header: "COOKIE DETAILS",
+          content: (
+            <>
+              <p>
+                <a href="/near/widget/NearOrg.CookieDetails/">See details on all cookies in use on near.org her.e</a>
```

Comment:
@jackson-harris-iii  Please use `${REPL_ACCOUNT}` instead of `near`. To support cross environment deploys.  

Also looks like there's a typo on `here`